api/config/example_config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "AWSBucketRegion": "us-east-1",
+  "AWSCloudwatchRegion": "us-east-1",
```

Comment:
Looks like some of these vars were removed, so probably shouldn't be in the example file

Reply:
done